base_google_map/models/res_config_settings.py

```diff
@@ -0,0 +1,226 @@
+    ('ky', 'Kyrgyz'),
+    ('eu', 'Basque'),
+    ('lo', 'Lao'),
+    ('be', 'Belarusian'),
+    ('lv', 'Latvian'),
+    ('bn', 'Bengali'),
+    ('lt', 'Lithuanian'),
+    ('bs', 'Bosnian'),
+    ('mk', 'Macedonian'),
+    ('bg', 'Bulgarian'),
+    ('ms', 'Malay'),
+    ('my', 'Burmese'),
+    ('ml', 'Malayalam'),
+    ('ca', 'Catalan'),
+    ('mr', 'Marathi'),
+    ('zh', 'Chinese'),
+    ('mn', 'Mongolian'),
+    ('zh-CN', 'Chinese (Simplified)'),
+    ('ne', 'Nepali'),
+    ('zh-HK', 'Chinese (Hong Kong)'),
+    ('no', 'Norwegian'),
+    ('zh-TW', 'Chinese (Traditional)'),
+    ('pl', 'Polish'),
+    ('hr', 'Croatian'),
+    ('pt', 'Portuguese'),
+    ('cs', 'Czech'),
+    ('pt-BR', 'Portuguese (Brazil)'),
+    ('da', 'Danish'),
+    ('pt-PT', 'Portuguese (Portugal)'),
+    ('nl', 'Dutch'),
+    ('pa', 'Punjabi'),
+    ('en', 'English'),
+    ('ro', 'Romanian'),
+    ('en-AU', 'English (Australian)'),
+    ('ru', 'Russian'),
+    ('en-GB', 'English (Great Britain)'),
+    ('sr', 'Serbian'),
+    ('et', 'Estonian'),
+    ('si', 'Sinhalese'),
+    ('fa', 'Farsi'),
+    ('sk', 'Slovak'),
+    ('fi', 'Finnish'),
+    ('sl', 'Slovenian'),
+    ('fil', 'Filipino'),
+    ('es', 'Spanish'),
+    ('fr', 'French'),
+    ('es-419', 'Spanish (Latin America)'),
+    ('fr-CA', 'French (Canada)'),
+    ('sw', 'Swahili'),
+    ('gl', 'Galician'),
+    ('sv', 'Swedish'),
+    ('ka', 'Georgian'),
+    ('ta', 'Tamil'),
+    ('de', 'German'),
+    ('te', 'Telugu'),
+    ('el', 'Greek'),
+    ('th', 'Thai'),
+    ('gu', 'Gujarati'),
+    ('tr', 'Turkish'),
+    ('iw', 'Hebrew'),
+    ('uk', 'Ukrainian'),
+    ('hi', 'Hindi'),
+    ('ur', 'Urdu'),
+    ('hu', 'Hungarian'),
+    ('uz', 'Uzbek'),
+    ('is', 'Icelandic'),
+    ('vi', 'Vietnamese'),
+    ('id', 'Indonesian'),
+    ('zu', 'Zulu'),
+    ('it', 'Italian'),
+]
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    @api.model
+    def get_region_selection(self):
+        country_ids = self.env['res.country'].search([])
+        values = [(country.code, country.name) for country in country_ids]
+        return values
+
+    google_maps_view_api_key = fields.Char(string='Google Maps View Api Key')
+    google_maps_lang_localization = fields.Selection(
+        selection=GMAPS_LANG_LOCALIZATION,
+        string='Google Maps Language Localization')
+    google_maps_region_localization = fields.Selection(
+        selection=get_region_selection,
+        string='Google Maps Region Localization')
+    google_maps_theme = fields.Selection(
+        selection=[('default', 'Default'),
+                   ('aubergine', 'Aubergine'),
+                   ('night', 'Night'),
+                   ('dark', 'Dark'),
+                   ('retro', 'Retro'),
+                   ('silver', 'Silver')],
+        string='Map theme')
+    google_maps_places = fields.Boolean(string='Places', default=True)
+    google_maps_geometry = fields.Boolean(string='Geometry', default=True)
```

Comment:
I expected to see here additional fields, proposing the User to install the other options: web_widget_google_marker_icon_picker and web_widget_google_address_autocomplete.